python/pyspark/streaming/tests.py

```diff
@@ -41,8 +41,8 @@
 
 class PySparkStreamingTestCase(unittest.TestCase):
 
-    timeout = 4  # seconds
-    duration = .2
```

Comment:
It's batch duration. These tests became flaky when we change the duration from 1 to 0.2, I'd like to try  to change it to 0.5. If they are still flaky, then move to 1 second.